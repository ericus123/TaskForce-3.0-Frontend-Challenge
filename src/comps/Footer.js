import { Navbar, Container } from "react-bootstrap";
import "./styles.scss";

const Footer = () => {
    return (
        <Navbar  collapseOnSelect expand="sm"className="footer">
            <Container>
              
                <Navbar.Collapse className="justify-content-start" style={{color:"white"}} >
                    <span style={{color:"#ccc"}}>Developed by </span><a style={{color:"white"}} className="text-decoration-none" rel="noreferrer" href="https://www.amanieric.com/" target="_blank"> &nbsp; AMANI Eric</a>
                </Navbar.Collapse>
                <Navbar.Collapse className="justify-content-end" style={{color:"white"}}>
                    <span style={{color:"#ccc"}}>Designed by </span> &nbsp;<a className="text-decoration-none" style={{color:"white"}} rel="norefferrer noreferrer" href="https://awesomity.rw/" target="_blank"> Awesomity Lab </a> </Navbar.Collapse>
            </Container>
        </Navbar>
    );
};
export default Footer;