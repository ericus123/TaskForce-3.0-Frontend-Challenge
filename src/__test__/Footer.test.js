/* eslint-disable no-undef */
import React from "react";
import { shallow,configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Footer from "../comps/Footer";
import toJson from "enzyme-to-json";


configure({adapter: new Adapter()});
it("renders footer without crashing", () => {
    let wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
});

