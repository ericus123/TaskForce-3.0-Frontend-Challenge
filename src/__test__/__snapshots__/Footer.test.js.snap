// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer without crashing 1`] = `
<Navbar
  className="footer"
  collapseOnSelect={true}
  expand="sm"
  variant="light"
>
  <Container
    fluid={false}
  >
    <NavbarCollapse
      className="justify-content-start"
      style={
        Object {
          "color": "white",
        }
      }
    >
      <span
        style={
          Object {
            "color": "#ccc",
          }
        }
      >
        Developed by 
      </span>
      <a
        className="text-decoration-none"
        href="https://www.amanieric.com/"
        rel="noreferrer"
        style={
          Object {
            "color": "white",
          }
        }
        target="_blank"
      >
           AMANI Eric
      </a>
    </NavbarCollapse>
    <NavbarCollapse
      className="justify-content-end"
      style={
        Object {
          "color": "white",
        }
      }
    >
      <span
        style={
          Object {
            "color": "#ccc",
          }
        }
      >
        Designed by 
      </span>
        
      <a
        className="text-decoration-none"
        href="https://awesomity.rw/"
        rel="norefferrer noreferrer"
        style={
          Object {
            "color": "white",
          }
        }
        target="_blank"
      >
         Awesomity Lab 
      </a>
       
    </NavbarCollapse>
  </Container>
</Navbar>
`;
